responsibleai/tests/databalanceanalysis/conftest.py

```diff
@@ -1,13 +1,15 @@
 # Copyright (c) Microsoft Corporation
 # Licensed under the MIT License.
 
-import copy
-
 import pandas as pd
 import pytest
 
 from ..common_utils import create_adult_income_dataset
 
+SYNTHETIC_DATA_LABEL = "Label"
+GENDER = "Gender"
```

Comment:
Technically, "adult" only has sex, not gender. Depending on where the data comes from that may not be represented correctly (sadly).

Reply:
Cleaning this up by updating the constant names and adding a small comment!

```
# Constants representing columns in the synthetic_data fixture
SYNTHETIC_DATA_LABEL = "Label"
SYNTHETIC_DATA_GENDER = "Gender"
SYNTHETIC_DATA_ETHNICITY = "Ethnicity"
```

Reply:
It's synthetic? Adult is a real dataset, isn't it?

Reply:
These constants are specifying col names for the synthetic data dataset in conftest.py!

Should I move these constants closer to the dataset instead of having at the very top?

Reply:
Oh I'm just surprised the adult dataset is synthetic. Are you quite sure?
https://archive.ics.uci.edu/ml/datasets/Adult

Reply:
Sorry for the confusion 😁 the constants are being used to create this synthetic dataset in conftest:
https://github.com/microsoft/responsible-ai-toolbox/blob/1c6532c696e363e5c8499be1b3967861dc6f8b39/responsibleai/tests/databalanceanalysis/conftest.py#L85